### PR TITLE
Hourlygraph

### PR DESCRIPTION
**Pull request by Cortesn**

### CODE REVIEW

**Review by stuttskl — DISMISSED** [review]
Looks great!

File: src/Components/Graphs/Hourgraph.js [hunk 1]
```diff
@@ -1,127 +1,56 @@
...
-                <CartesianGrid strokeDasharray="3 3" />
-                <XAxis dataKey="time" />
-                <YAxis label={{ value: '°F', angle: -90, position: 'insideLeft' }} />
-                <Tooltip />
-                <Line type="monotone" dataKey="temperature" stroke="#8884d8" />
-            </LineChart>
+            <Grid container spacing={2} justify="center">
+                <Grid item xs={12}>
+                <Typography variant="h4" align="center" display="block">
+                    24Hr Forecast
```
Comment (tdraxler) [inline src/Components/Graphs/Hourgraph.js]: I suggest changing this to "24 Hour Forecast" (because of its prominent position in the app)

**Review by tdraxler — DISMISSED** [review]
Fantastic work, Nick! Looks polished, I like the loading animations, and the date/time conversion is perfect.
I'd suggest tweaking the title of the 24-hour graph before merging, but other than that it's looking great.

File: src/Components/Graphs/Hourgraph.js (hunk 1 above)
Comment (Cortesn) [reply src/Components/Graphs/Hourgraph.js]: Updated graph title with latest commit

**Review by tdraxler — APPROVED** [review]
Great job!

**Review by mswarts — APPROVED** [review]

**Pull request merged**